components/datadog/agent/kubernetes_helm.go

```diff
@@ -204,6 +204,12 @@ func buildLinuxHelmValues(baseName, agentImagePath, agentImageTag, clusterAgentI
 			"apiKeyExistingSecret": pulumi.String(baseName + "-datadog-credentials"),
 			"appKeyExistingSecret": pulumi.String(baseName + "-datadog-credentials"),
 			"checksCardinality":    pulumi.String("high"),
+			"namespaceLabelsAsTags": pulumi.Map{
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should these be params configurable from a test instead of fixed strings?

Reply:
They (team and email) are tag names. The values are passed from nginx namespace. It is not convenient to set dummy apps namespace from a test. So we move them to test-infra-def.